src/main/java/org/opensearch/security/securityconf/impl/v7/ConfigV7.java

```diff
@@ -478,4 +480,51 @@ public String toString() {
 
     }
 
+    public static class OnBehalfOf {
+        @JsonProperty("enabled")
+        private Boolean oboEnabled;
```

Comment:
I think the null check in `getOboEnabled` can be removed if you set this to `private boolean oboEnabled = true;`

FYI primitives like boolean will never be null. Even if you declare `private boolean oboEnabled;` it will default to a value of `false`. Can this be changed to use the primitive `boolean`?

Reply:
Fixed. Thanks Craig! I get it now. Yes this is much cleaner.